e_dashboard.php

```diff
@@ -70,7 +70,7 @@ function renderChart()
 				'pointColor '		=> "rgba(151,187,205,1)",
 				'pointStrokeColor'	=> "#fff",
 				'data'				=> $dbdata['B'],
-				'title'				=> 'Payed'
+				'title'				=> ''.LAN_VSTORE_CUSM_033.''
```

Comment:
This is looking much better.. I can see what changed now.  :-)
btw. 
`''.LAN_VSTORE_CUSM_033.''` should be changed to `LAN_VSTORE_CUSM_033` 

.. the same for the others like it.

Reply:
Yep thought so ( more a case of being sure instead of error), anyway i have to recompile, will take it as noted.